packages/integration-tests/suites/replay/privacy/test.ts

```diff
@@ -0,0 +1,226 @@
+               'id': 2,
+               'name': 'html',
+               'publicId': '',
+               'systemId': '',
+               'type': 1,
+             },
+              {
+               'attributes':  {},
+               'childNodes':  [
+                  {
+                   'attributes':  {},
+                   'childNodes':  [
+                      {
+                       'attributes':  {
+                         'charset': 'utf-8',
+                       },
+                       'childNodes':  [],
+                       'id': 5,
+                       'tagName': 'meta',
+                       'type': 2,
+                     },
+                   ],
+                   'id': 4,
+                   'tagName': 'head',
+                   'type': 2,
+                 },
+                  {
+                   'id': 6,
+                   'textContent': '\n  ',
+                   'type': 3,
+                 },
+                  {
+                   'attributes':  {},
+                   'childNodes':  [
+                      {
+                       'id': 8,
+                       'textContent': '\n    ',
+                       'type': 3,
+                     },
+                      {
+                       'attributes':  {
+                         'aria-label': 'Click me',
+                         'onclick': "console.log('Test log')",
+                       },
+                       'childNodes':  [
+                          {
+                           'id': 10,
+                           'textContent': '***** **',
+                           'type': 3,
+                         },
+                       ],
+                       'id': 9,
+                       'tagName': 'button',
+                       'type': 2,
+                     },
+                      {
+                       'id': 11,
+                       'textContent': '\n    ',
+                       'type': 3,
+                     },
+                      {
+                       'attributes':  {},
+                       'childNodes':  [
+                          {
+                           'id': 13,
+                           'textContent': '**** ****** ** ****** ** *******',
+                           'type': 3,
+                         },
+                       ],
+                       'id': 12,
+                       'tagName': 'div',
+                       'type': 2,
+                     },
+                      {
+                       'id': 14,
+                       'textContent': '\n    ',
+                       'type': 3,
+                     },
+                      {
+                       'attributes':  {
+                         'data-sentry-unmask': '',
+                       },
+                       'childNodes':  [
+                          {
+                           'id': 16,
+                           'textContent': 'This should be unmasked due to data attribute',
```

Comment:
This should break when we merge and release https://github.com/getsentry/rrweb/pull/40